File: app/src/main/java/id/akakom/bimo/shareloc/module/Activity/ShareLocActivity.java
package id.akakom.bimo.shareloc.module.Activity;

import android.content.Context;
import android.os.Bundle;

import id.akakom.bimo.shareloc.R;

import lib.gmsframeworkx.Activity.ActivityGeneral;
import uk.co.chrisjenx.calligraphy.CalligraphyConfig;
import uk.co.chrisjenx.calligraphy.CalligraphyContextWrapper;

public class ShareLocActivity extends ActivityGeneral {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        CalligraphyConfig.initDefault(new CalligraphyConfig.Builder()
                .setDefaultFontPath("fonts/SanFransisco-Regular.otf")
                .setFontAttrId(R.attr.fontPath)
                .build()
        );
    }

    @Override
    protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase));
    }
}

File: app/src/main/java/id/akakom/bimo/shareloc/module/Activity/SharelocActivity.kt
package id.akakom.bimo.shareloc.module.Activity

import android.content.Context
import android.os.Bundle
import id.akakom.bimo.shareloc.R
import lib.gmsframeworkx.Activity.ActivityGeneral
import uk.co.chrisjenx.calligraphy.CalligraphyConfig
import uk.co.chrisjenx.calligraphy.CalligraphyContextWrapper


class ShareLocActivity : ActivityGeneral() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        CalligraphyConfig.initDefault(
            CalligraphyConfig.Builder()
                .setDefaultFontPath("fonts/SanFransisco-Regular.otf")
                .setFontAttrId(R.attr.fontPath)
                .build()
        )
    }

    override fun attachBaseContext(newBase: Context) {
        super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase))
    }
}